app/src/main/java/com/example/parkingsystem/mvp/presenter/ParkingPresenter.java

```diff
@@ -0,0 +1,25 @@
+package com.example.parkingsystem.mvp.presenter;
+
+
+import com.example.parkingsystem.mvp.contract.ParkingContract;
+import com.example.parkingsystem.mvp.model.ParkingModel;
+import com.example.parkingsystem.mvp.view.ParkingView;
+
+public class ParkingPresenter implements ParkingContract.ParkingPresenter {
+
+    private ParkingModel model;
+    private ParkingView view;
+
+
+    public ParkingPresenter(ParkingModel model, ParkingView view) {
```

Comment:
here you should have the contract types too

Reply:
Done!